wweval/R/ms_fig3.R

```diff
@@ -319,11 +316,12 @@ make_fig3_crps_underlay_fig <- function(scores,
     ) +
     get_plot_theme(
       x_axis_dates = TRUE,
-      y_axis_title_size = 8
+      y_axis_title_size = 8,
+      y_axis_text_size = 6
     ) +
     scale_y_continuous(
       # don't expand y scale at the lower end
-      expand = expansion(mult = c(0, 0.05))
+      limits = c(0, max_crps + 0.05)
```

Comment:
This is the problem. y-axis truncation should happen in `coord_cartesian`, not `scale_y_continuous`, unless you actually want to truncate the data (but I don't think you do).

Reply:
Open Q whether we want these to have a shared y-axis or not -- the current version of the manuscript has a "free" y axis for each horizon, but I think it might be reasonable to consider making them all standardized.

Otherwise I think we can just get rid of this.